applets/005-netperf/applet.py

```diff
@@ -29,7 +30,7 @@ def netserver_start(x, host_name, host_ip, host_user, netserver_port):
 def main(x, conf, args):
 
     if not len(args) == 7:
-        x.p.msg("wrong usage. use: source dest mode:[options] sport:[port] "
+        x.p.msg("wrong usage. use: source dest id:[id] sport:[port] "
```

Comment:
Better to name the dest option more explicit? Probably more self-commenting

E.g.

alpha

vs.

dest:alpha